Automattic/wp-calypso
# Writing Prompts - Add navaigate buttons to my home writing prompt card

#### Proposed Changes

* Update promo card on My Home screen with new design that allows customers to toggle through writing prompts and see a link to all submissions:
<img width="574" alt="image" src="https://user-images.githubusercontent.com/5560595/212082865-4d09d513-c79e-43db-841d-a6910fb0bbdc.png">
<img width="713" alt="cleanshot-2023-01-10-at-07 46 03402x" src="https://user-images.githubusercontent.com/5560595/212082856-89d30727-78d1-44e7-acea-09479837b6f3.png">

### Changes
* This PR adds buttons to allow the user to chose up to 10 prompts from today (should we allow more?).
* The buttons behaviour will only allow you to move back and forth between the group of prompts you retrieved from the blogging prompts API endpoint.
* The home card will also show a list of avatars for users that have responded to this prompt - the API returns 3 avatar URLs.
* The home card links to the readers `#dailyprompt-{prompt-id}` tag to allow the users to see other answers to this prompt.

#### Testing Instructions

* Apply patch and go to `http://calypso.localhost:3000/home/{TEST SITE SLUG}`
* Test my home card works as expected



Related to https://github.com/Automattic/wp-calypso/issues/71997
